tox.ini

```diff
@@ -5,7 +5,7 @@ skipsdist = True
 [testenv]
 passenv = 
     CODECOV_* TOXENV CI TRAVIS TRAVIS_*
-    DJANGO_SETTINGS_MODULE DATABASE_URL
+    DATABASE_URL SECRET_KEY
```

Comment:
Are you sure that just passing `DJANGO_SETTINGS_MODULE` wouldn't be enough? That sounds like it would work, since it would define the `SECRET_KEY`.

Reply:
Yeah that's also my first thought. But it is required because ```collectstatic``` won't use pytest's test settings. :smile: